src/sctools/fastq.py

```diff
@@ -160,41 +160,83 @@ def __iter__(self):
 EmbeddedBarcode = namedtuple('Tag', ['start', 'end', 'sequence_tag', 'quality_tag'])
 
 
+def extract_barcode(record, eb):
```

Comment:
I think it would be clearer to avoid short abbreviations for variable names where possible. In this case I find `eb` confusing because I thought it stood for `extracted_barcode` but it's actually `embedded_barcode`

Reply:
Did it to avoid multi-lining some of the commands; turns out it was fine with the full name, too. :)